venia-integration-tests/src/tests/wishList/integrationTests/verifyMultipleWishlistFeatures.spec.js

```diff
@@ -0,0 +1,227 @@
+import {
+    accountAccess as accountAccessFixtures,
+    homePage as homePageFixtures,
+    wishlist as wishlistFixtures,
+    categoryPage as categoryPageFixtures,
+    productPage as productPageFixtures,
+    graphqlMockedCalls as graphqlMockedCallsFixtures
+} from '../../../fixtures';
+import {
+    myAccountMenu as myAccountMenuAssertions,
+    wishlist as wishlistAssertions,
+    categoryPage as categoryPageAssertions,
+    productPage as productPageAssertions
+} from '../../../assertions';
+import {
+    wishlistPage as wishlistPageActions,
+    categoryPage as categoryPageActions,
+    productPage as productPageActions
+} from '../../../actions';
+const {
+    firstName,
+    lastName,
+    accountEmail,
+    accountPassword
+} = accountAccessFixtures;
+const { homePage } = homePageFixtures;
+const { assertCreateAccount } = myAccountMenuAssertions;
+const { wishistRoute } = wishlistFixtures;
+const { createWishlist } = wishlistPageActions;
+const {
+    assertEmptyWishlistPage,
+    assertEmptyWishlistExists,
+    assertCreateWishlistLink,
+    assertProductInWishlist
+} = wishlistAssertions;
+const { categoryTops, productCarinaCardigan } = categoryPageFixtures;
+const {
+    addProductToWishlistFromCategoryPage,
+    createWishlistViaDialog
+} = categoryPageActions;
+const { assertWishlistSelectedProductOnCategoryPage } = categoryPageAssertions;
+const {
+    productAugustaEarningsUrl,
+    productAugustaEarningsName
+} = productPageFixtures;
+const {
+    addProductToWishlistFromProductPage,
+    addProductToExistingWishlistFromDialog
+} = productPageActions;
+const { assertProductSelectIndicator } = productPageAssertions;
+const {
+    getCustomerWishlistCall,
+    getMultipleWishlistConfigCall,
+    getWishlistDialogDataCall,
+    getWishlistitemsForLocalFieldsCall,
+    hitGraphqlPath,
+    getWishlistConfigForGalleryCall,
+    getWishlistConfigForProductPageCall
+} = graphqlMockedCallsFixtures;
+
+// TODO add tags CE, EE to test to filter and run tests as needed
+describe('verify single wishlist basic features', () => {
+    it('user should be able to add and remove products from wishlist', () => {
+        cy.visitPage(homePage);
+
+        cy.openLoginDialog();
+        cy.createAccount(
+            accountAccessFixtures.firstName,
+            lastName,
+            accountEmail,
+            accountPassword
+        );
+
+        assertCreateAccount(firstName);
```

Comment:
I'd prefer only to `assert` the things that this specific case is testing for. 

If creating an account doesn't work, that should show up as a failure within the create account section of tests.

Reply:
agreed, good call. I put it here to make sure cypress waits for account creation is complete before it goes to next step. that is visiting `/wishlist` route in this case.   I have added comment above it, but let me know if you think of any better approach here.